web/tests/password.test.cjs

```diff
@@ -73,4 +75,12 @@ run_test("basics w/progress bar", () => {
     assert.equal($bar.added_class, "bar-danger");
     warning = password_warning(password, password_field(6));
     assert.equal(warning, 'Repeated characters like "aaa" are easy to guess.');
+
+    // Test a password that's longer than the configured limit.
+    password = "hfHeo34FksdBChjeruShJ@sidfgusd";
```

Comment:
I redid this test to use a much shorter password. This avoids this test spending almost a 1s (10% of total runtime of the suite) just computing the strength of a giant password.